icd/api/include/vk_shader.h

```diff
@@ -34,8 +34,10 @@
 
 namespace Pal { enum class ResourceMappingNodeType : Pal::uint32; }
 
+#if LLPC_CLIENT_INTERFACE_MAJOR_VERSION < 50
```

Comment:
@amdrexu could you review this change? in internal branch, this definition is completely removed instead of wrapped with #if/#endif

Reply:
Should keep #if and #endif. The internal branch is bumped to 50 so it could fix this but the solution is not good.